rir/src/interpreter/interp.cpp

```diff
@@ -1072,6 +1074,9 @@ static void cachedSetVar(SEXP val, SEXP env, Immediate idx, Context* ctx,
     UNPROTECT(1);
 }
 
+#pragma GCC diagnostic push
```

Comment:
Will this cause problems for clang?

Reply:
apparently not. I think they support most GCC pragmas that people use....